test/acl.test.js

```diff
@@ -393,20 +393,18 @@ describe('security ACLs', function() {
 });
 
 describe('access check', function() {
-  var app;
-  before(function() {
-    app = loopback();
-    app.set('remoting', { errorHandler: { debug: true, log: false }});
-    app.use(loopback.rest());
-    app.enableAuth();
-    app.dataSource('test', { connector: 'memory' });
```

Comment:
@bajtos : In the previous commit, IIRC, _(can't quote as rebased)_, I had kept the `before()` part, but I was advised to remove it as there is only one case in that `describe()` block.

Do you still recommend removing it?